gitPractice/1-gitHomework.js

```diff
@@ -0,0 +1,45 @@
+//? practice to undrestand difference between const and let.
+//const and let are variable, on the other hand const is better than let because const variable won't be reasigned however let could be reasigned.
+// In conclusion:
+// "Var" variable can be updated and re-declared.don't need initialized.
+// "let" variable can be updated but not re-declared. don't need initialized.
+// "const" variable neither be update and re-declared. needs initialized.
+
+//?practice git and read the codeyourfuture git syllabus.
+//I read the git from codeYourFuture syllabus and also I watched the https://www.youtube.com/watch?v=BCQHnlnPusY and https://www.youtube.com/watch?v=iv8rSLsi1xo
+// Now I know how I can make a repo and how I can commit and add more commit, eventhough we could do the same in VS, git and gitHub.
```

Comment:
@RominaBideh  Great, did you read the whole git section of the syllabus?

Reply:
Yes, I did. However, I need to recap them.